src/binder/bind_node_visitor.cpp

```diff
@@ -61,10 +61,27 @@ void BindNodeVisitor::Visit(parser::SelectStatement *node, parser::ParseResult *
   BINDER_LOG_TRACE("Gathering select columns...");
   for (auto &select_element : node->GetSelectColumns()) {
     if (select_element->GetExpressionType() == parser::ExpressionType::STAR) {
-      context_->GenerateAllColumnExpressions(parse_result, &new_select_list);
+      context_->GenerateAllColumnExpressions(parse_result, &new_select_list, "");
       continue;
     }
 
+    // Check if there is a table qualifier with a star expression
+    if (select_element->GetExpressionType() == parser::ExpressionType::COLUMN_VALUE) {
+      auto *expr = static_cast<parser::ColumnValueExpression *>(select_element.Get());
+      std::tuple<catalog::db_oid_t, catalog::table_oid_t, catalog::Schema> tuple;
+      std::string table_name = expr->GetTableName();
+      std::string col_name = expr->GetColumnName();
+      if (col_name == "*") {
```

Comment:
I am surprised, I thought we had a star expression.

Reply:
There is a generic `StarExpression`, but not for table qualifiers. This case is simply not handled at all.